bindings/python/tests/conftest.py

```diff
@@ -0,0 +1,58 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import re
+
+import opendal
+import pytest
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+# Define the regex pattern to match "OPENDAL_XXX_TEST"
+pattern = r'^OPENDAL_(\w+)_TEST$'
+regex = re.compile(pattern)
+
+
+@pytest.fixture
```

Comment:
For fixture, I expect the following things:

- The fixture should be shared across the whole test suite. The opeator should only be init once. Maybe we need to have a `class TestSuite`?
- Based on the class level shared fixture, we can have `class TestS3`, `class TestFs` so users can pick the services they want to run at the same time? For example `pytest -k TestS3`?